addons/website/models/theme_models.py

```diff
@@ -275,6 +275,9 @@ class IrUiView(models.Model):
     theme_template_id = fields.Many2one('theme.ir.ui.view', copy=False)
 
     def write(self, vals):
+        # We only do this if we are in a module update state
+        if not self.pool._init:
+            return super(IrUiView, self).write(vals)
```

Comment:
```suggestion
            return super().write(vals)
```
Python 3 :+1:

Reply:
Thank you. I just wrote it to look like the code below, no other reason. I will fix it.